### PR TITLE
Update I/O of Search command, Add tests for ListPlacesCommand and SearchCommand

### PR DESCRIPTION
**Pull request by tengkianen**

### CODE REVIEW

**Review by weishuangtan — APPROVED** [review]
lgtm!

File: src/main/java/seedu/trippie/command/SearchCommand.java [hunk 1]
```diff
@@ -32,12 +32,10 @@ public boolean isExit() {
     public void execute(Ui ui, TrippieData trippieData) {
         List<Place> list = trippieData.getCurrentTrip().getPlaceListObject().getPlaceList();
         int size = list.size();
-        String listPlurality = size > 1 ? "are" : "is";
-        String placePlurality = size > 1 ? "places" : "place";
         if (size == 0) {
             System.out.println("There is currently nothing in your place list. Why not add one?");
         } else {
-            System.out.printf("Here %s the matching %s in your list:%n", listPlurality, placePlurality);
+            System.out.printf("Here is your search result:%n");
```
Comment (felixhalim) [inline src/main/java/seedu/trippie/command/SearchCommand.java:38]: The search result may be more than 1, the previous method to determine plurality can handle this.

File: src/test/java/seedu/trippie/command/ListPlacesCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        Trip newTrip = new Trip(trippieData.getTripList().size(), "Singapore", df.parse("11-11-2020"));
+        newTrip.getExpenseListObject().setForExValue(Float.parseFloat("100"));
+        newTrip.getExpenseListObject().setCurrencyAbbreviation("SGD");
+        newTrip.getExpenseListObject().setBudgetValue(Float.parseFloat("1000"));
+        int index = trippieData.getTripList().size();
+        trippieData.getTripList().add(newTrip);
+        trippieData.setCurrentTripFromIndex(index);
+    }
+
+}
```
Comment (felixhalim) [inline src/test/java/seedu/trippie/command/ListPlacesCommandTest.java:121]: Add `newline` here

File: src/test/java/seedu/trippie/command/SearchCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        SimpleDateFormat df = new SimpleDateFormat("dd-MM-yyyy");
+        Trip newTrip = new Trip(trippieData.getTripList().size(), "Singapore", df.parse("11-11-2020"));
+        newTrip.getExpenseListObject().setForExValue(Float.parseFloat("100"));
+        newTrip.getExpenseListObject().setCurrencyAbbreviation("SGD");
+        newTrip.getExpenseListObject().setBudgetValue(Float.parseFloat("1000"));
+        int index = trippieData.getTripList().size();
+        trippieData.getTripList().add(newTrip);
+        trippieData.setCurrentTripFromIndex(index);
+    }
+}
```
Comment (felixhalim) [inline src/test/java/seedu/trippie/command/SearchCommandTest.java:108]: Another `newline` here

**Review by felixhalim — COMMENTED** [review]
Great job in implementing a test for Search and ListPlaces command. I have added some comments for improvement, please kindly refer to that.

**Review by felixhalim — APPROVED** [review]

**Pull request merged**